### PR TITLE
Removed GoldenLayout

### PR DESCRIPTION
**Pull request by ivanbatic**
This PR removes GoldenLayout in favor of custom implementation of our own basic necessities.
This also removes jQuery from our depencency list.
Package.json is updated as well with some dependencies moved to the dev section.
localhost_9000

### CODE REVIEW

File: src/app/components/placeholders/file-editor/file-editor-placeholder.component.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ require("./file-editor-placeholder.component.scss");
     selector: "file-editor-placeholder",
     template: `
        <div class="full-size-table-display">
-            
+                yoyoyoyo
```
Comment (mayacoda) [inline src/app/components/placeholders/file-editor/file-editor-placeholder.component.ts]: 👍

File: src/app/components/layout/layout.component.ts [hunk 2]
```diff
@@ -28,27 +27,17 @@ export class LayoutComponent implements OnInit {
     private tree: ElementRef;
 
     @Input()
-    private treeSize = 0.35;
+    private treeSize = .2;
```
Comment (mayacoda) [inline src/app/components/layout/layout.component.ts]: Would it be possible to have a max-width for the file tree? I messed with the CSS a little and failed to produce any desirable results.

File: src/app/components/workbox-tab.component.ts/workbox-tab.component.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import {Component} from "@angular/core";
```
Comment (matesk8) [inline src/app/components/workbox-tab.component.ts/workbox-tab.component.ts]: Is this just Github, or is this in a directory called "workbox-tab.component.ts" ?

File: src/assets/sass/main.scss [hunk 4]
```diff
@@ -9,8 +9,6 @@
 @import "vectors";
 @import "forms";
 
-@import "themes/seven-bridges";
```
Comment (mayacoda) [inline src/assets/sass/main.scss]: Should this file be deleted?

File: src/app/components/workbox-tab.component.ts/workbox.component.ts [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+            this.activeTab = this.tabs[this.tabs.length - 1];
+        }));
+
+    }
+
+    /**
+     * Removes a tab by index
+     * @param index
+     */
+    public removeTab(index: Number) {
```
Comment (mayacoda) [inline src/app/components/workbox-tab.component.ts/workbox.component.ts]: This logic doesn't work for removing tabs that are in the middle. I have tabs A, B, C. Click close for B, A closes, left with B, C  with B in focus.

Comment (ivanbatic) [inline src/app/components/workbox-tab.component.ts/workbox.component.ts]: Good catch, fixed.

File: src/app/components/workbox-tab.component.ts/workbox-tab.component.ts (hunk 3 above)
Comment (ivanbatic) [inline src/app/components/workbox-tab.component.ts/workbox-tab.component.ts]: Fixed

File: src/assets/sass/main.scss (hunk 4 above)
Comment (ivanbatic) [inline src/assets/sass/main.scss]: Done.

File: src/app/components/layout/layout.component.ts (hunk 2 above)
Comment (ivanbatic) [inline src/app/components/layout/layout.component.ts]: Done.

**Pull request merged**